tox.ini

```diff
@@ -87,4 +87,4 @@ commands =
     pydocstyle {toxinidir}/celery
 
 [testenv:isort-check]
-commands = isort --project celery --order-by-type -rc -c {toxinidir}/celery {toxinidir}/t
+commands = isort --project celery --diff --order-by-type -rc -c {toxinidir}/celery {toxinidir}/t
```

Comment:
Why this change? If needed we can discuss it in a separate PR.

Reply:
@thedrow it is not directly related, but it was quite difficult to understand why the `isort-check` build job was failing. By providing the diff, there is actual feedback to the developer about the changes that `isort` would perform in order to pass.

I agree that however small the change, optimally we should apply it separately. Let me know what works best in your opinion.

Reply:
Please pull this change to a different PR as it is not crucial for us to merge in order to release 4.2.

Reply:
Opened #4469 for this purpose.

Reply:
@thedrow I removed the commit and rebased.